.github/workflows/ci.yml

```diff
@@ -130,8 +130,8 @@ jobs:
     name: Pytest Ubuntu
     strategy:
       matrix:
-        # TODO(#3800): remove 3.6 when Colab switches to 3.7
```

Comment:
we can remove this TODO for now, we are stuck on 3.6 compatibility until Google upgrades to 3.7.

Reply:
removed it